examples/catalog.json

```diff
@@ -15,12 +15,6 @@
       "href": "./modis-006/catalog.json",
       "type": "application/json",
       "title": "MODIS, version 006"
-    },
-    {
-      "rel": "child",
-      "href": "./modis-061/catalog.json",
-      "type": "application/json",
-      "title": "MODIS, version 061"
```

Comment:
Is there a reason we're removing the 061 catalog example?

Reply:
Nope. Side effect of not supporting footprints for 006 data. I think I need to work something in that handles that without blowing up (raises an error right now).

Reply:
Now logging a warning for unsupported collections when creating catalogs/collections via the CLI.